services/openuserjs/openuserjs-downloads.service.js

```diff
@@ -0,0 +1,27 @@
+import { renderDownloadsBadge } from '../downloads.js'
+import BaseOpenUserJSService from './openuserjs-base.js'
+
+export default class OpenUserJSDownloads extends BaseOpenUserJSService {
+  static category = 'downloads'
+  static route = { base: 'openuserjs', pattern: 'dt/:username/:scriptname' }
```

Comment:
Yup, leave the route as-is for now and I'll follow up with a resolution one way or another. I believe using the download nomenclature in the route is intentional but it's been a while since we last discussed as time and my memory is a bit fuzzy on where we landed

Reply:
We'll likely continue discussing the specifics of this offline, but for now I think we've reached a sufficient consensus that it's fine to go with this route as is. We can always add redirectors (our mechanism for updating routes without impacting existing users) for these badges down the road if we end up deciding to pivot on the route conventions.

I'll try to make time over the weekend to do a final pass over the subsequent updates

Reply:
Actually, I see after reading through the previously linked issues that the maintainers of the upstream service still have some concerns, so we'll put this on hold for now until we figure out whether there's a viable path forward or whether we should just abandon this.